src/components/transactionResult/index.js

```diff
@@ -2,15 +2,15 @@ import React from 'react';
 import { PrimaryButtonV2, TertiaryButtonV2 } from '../toolbox/buttons/button';
 import Illustration from '../toolbox/illustration';
 
-import styles from './votingResult.css';
+import styles from './transactionResult.css';
 
 const getErrorReportMailto = (error) => {
```

Comment:
I think we are using this in other files too, so maybe we can have this function in a utils file where we can use it and just pass the error message

Reply:
Done in https://github.com/LiskHQ/lisk-hub/pull/2085/commits/708ab8212f457057074fccc3d59c00d2506616e4